ui/src/wizard/StorageAccountNameStep.ts

```diff
@@ -38,13 +38,14 @@ export class StorageAccountNameStep<T extends types.IStorageAccountWizardContext
 
     private async validateStorageAccountName(client: StorageManagementClient, name: string): Promise<string | undefined> {
         name = name.trim();
-
+        
         if (!name || name.length < storageAccountNamingRules.minLength || name.length > storageAccountNamingRules.maxLength) {
             return localize('invalidLength', 'The name must be between {0} and {1} characters.', storageAccountNamingRules.minLength, storageAccountNamingRules.maxLength);
         } else if (name.match(storageAccountNamingRules.invalidCharsRegExp) !== null) {
             return localize('invalidChars', "The name can only contain lowercase letters and numbers.");
         } else {
-            const nameAvailabilityResult: StorageManagementModels.CheckNameAvailabilityResult = await client.storageAccounts.checkNameAvailability(name);
+            const type = "Microsoft.Storage/storageAccounts";
```

Comment:
We can extract this string out to a constant since we use it here and in StorageAccountListStep.ts